cosmwasm/cli-utils.js

```diff
@@ -100,13 +104,17 @@ const addInstantiateProposalOptions = (program) => {
 };
 
 const addExecuteProposalOptions = (program) => {
-    program.addOption(new Option('--msg <msg>', 'json encoded execute message'));
+    program.addOption(new Option('--msg <msg>', 'json encoded execute message').makeOptionMandatory(true));
 };
 
 const addParamChangeProposalOptions = (program) => {
     program.addOption(new Option('--changes <changes>', 'parameter changes'));
 };
 
+const addMigrateOptions = (program) => {
+    program.addOption(new Option('--msg <msg>', 'json encoded migration message').default('{}'));
```

Comment:
Avoid the default, make the user submit it explicitly

Reply:
thought about it, but I sometimes find myself forgetting about the right string which parses correctly to Rust `Empty`, so I thought making things easier by providing the default. The proposal would fail anyway if the message is not formatted correctly, but I don't have strong opinions about it.

Reply:
I'd prefer to be explicit. The user initiating the migration should ensure they know what the migration msg looks like.  You can mention it in the help text here, that if there is no migration msg, use `'{}'`.